repository_service_tuf_api/bootstrap.py

```diff
@@ -81,6 +84,25 @@ class Config:
         schema_extra = {"example": example}
 
 
+def _check_bootstrap_status(task_id, timeout):
+    time_timeout = time.time() + timeout
+
+    while True:
+        task = repository_metadata.AsyncResult(task_id)
+        if task.status == "SUCCESS":
+            return None
+        elif task.status == "FAILURE":
+            release_bootstrap_lock()
+            return None
+        else:
+            if time.time() > time_timeout:
+                task.revoke(terminate=True)
+                release_bootstrap_lock()
+                return None
```

Comment:
I think in all `return` statements you can just use `return` instead of `return None`.
You don't return anything else than `None`, so no sense in doing that.